hal_st/stm32fxxx/DigitalToAnalogPinStm.hpp

```diff
@@ -15,14 +19,37 @@ namespace hal
         : public DigitalToAnalogPinImplBase
     {
     public:
+#if defined(DAC_CHIPCONNECT_BOTH)
+        // clang-format off
+        static const struct Internal{} internal;
+        static const struct External{} external;
+        static const struct Both{} both;
+
+        // clang-format on
+
+        enum struct Channel : uint32_t
+        {
+            channel_1 = 1,
+            channel_2 = 2,
```

Comment:
```suggestion
            channel1 = 1,
            channel2 = 2,
```

Reply:
Since you're using an enum, why not use the default 0/1 values?

Reply:
It now is more in-line with the other uses of fetching the channels. (Else we would have to +1 these values in some constructors, and not in others)